.github/workflows/ci.yml

```diff
@@ -18,14 +18,25 @@ jobs:
     strategy:
       matrix:
         ruby: [2.2, 2.3, 2.4, 2.5, 2.6, 2.7]
-        activemodel: [3.2, 4.0, 4.1, 4.2, 5.0, 5.1, 5.2, 6.0]
+        activemodel: [3.2, 4.0, 4.1, 4.2, 5.0, 5.1, 5.2, 6.0, 6.1]
+        include:
```

Comment:
I've chosen `include` here rather than explicitly excluding ActiveModel 3.2 - 5.2 for Ruby 3.0